ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -89,60 +126,24 @@ def load_graph(self):
                 tf.import_graph_def(od_graph_def, name='')
         self.detection_graph = detection_graph
 
-    def get_classification(self, image):
+    def get_classification(self, img):
         """Determines the color of the traffic light in the image
 
         Args:
-            image (cv::Mat): image containing the traffic light
+            img (cv::Mat): image containing the crop of the traffic light
 
         Returns:
             int: ID of traffic light color (specified in styx_msgs/TrafficLight)
 
         """
-        # TODO implement light color prediction
-        #Parameters
-        MIN_AREA_FOR_STOP = 3
-        MIN_COMPACTNESS_FOR_BULB = 0.04
-
-        mask, area, perimeter = extract_red_area(image)
 
-        if area > MIN_AREA_FOR_STOP and area/(perimeter*perimeter)> MIN_COMPACTNESS_FOR_BULB:
-            state = TrafficLight.RED  # state = 'stop' >> id = 0
-            #print(area/(perimeter*perimeter))
+        if contains_red_light(img):
```

Comment:
Logic that detects area does not belong to the class and was moved out into a function